dts/bindings/base/zephyr,memory-region.yaml

```diff
@@ -5,7 +5,7 @@ description: Compatible for devices resulting in linker memory regions
 
 compatible: "zephyr,memory-region"
 
-include: base.yaml
+include: [base.yaml, "zephyr,memory-attr.yaml"]
```

Comment:
base is now redundant here?

Reply:
yes, but that is kinda on purpose. The point is that `zephyr,memory-attr.yaml` is only basically carrying a property so in 99% of the cases it will be included in a device binding where the `base.yaml` is already included